packages/core/src/components/bal-field/bal-field-hint/bal-field-hint.tsx

```diff
@@ -22,6 +27,8 @@ export class FieldHint {
    */
   @Prop() small = false
 
+  @State() isDesktop = balBreakpoints.isDesktop
```

Comment:
You need to listen to breakpoint changes use the listener we created.

  @ListenToBreakpoints()
  breakpointListener(breakpoints: BalBreakpoints): void {
    this.isDesktop = breakpoints.desktop
  }

Reply:
done